### DISCUSSION TITLE
Is this production ready?

### DISCUSSION THREAD

**Discussion by hiqsociety** [erthink/libmdbx#347] [Q&A] [answered]
Do you have a list of users using libmdbx?

**Comment by erthink** [comment, marked as answer]
1. Currently is no explicit list of users. On the other hand, libmdbx is used in [Positive Technologies](https://www.ptsecurity.com/) products (included in several Gartner quadrants) since 2017, and by [Erigon](https://erigon.tech) since 2021.
2. libmdbx is considered ready for production use and I pay a lot of attention to testing.
I believe that libmdbx is not only faster, but it definitely contains fewer bugs compared to LMDB.
However, in new and/or extreme scenarios, regressions ([for instance](https://github.com/erthink/libmdbx/issues/164)) are sometimes discovered which are then fixed.
So, I always recommend testing and verifying as much as possible.
3. No significant development of libmdbx is planned, as this requires breaking compatibility.
However, libmdbx will be supported (including the implementation of various optimizations and enhancements at least for the life cycle of Positive Technologies products, i.e. ~5 years.
Presumably, during this time, the code will be very well tested and stabilized.
Therefore, follow-up support will require very little effort and can be provided through small sponsorship and / or volunteers.
